infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -92,5 +95,8 @@
       "convert_self_signed": {{ to_le_convert_self_signed | bool | lower }},
       "renew_days_before_expiration": {{ to_le_renew_days_before_expiration | int }},
       "environment": "{{ to_le_environment }}"
+   },
+   "client_certificate_authentication" : {
+      "root_certificates_directory" : "{{ to_client_cert_root_directory }}"
```

Comment:
What creates `to_client_cert_root_directory`? How do we know it exists?

Reply:
It might not exist - for our purposes I handled it in ansible-pull. It already exists on all of our nightly boxes. I will handle this the same way when going to production.

Reply:
I added a conditional block to leave this out if the client-cert authentication is disabled.